shiny/Data-Ingest/app.R

```diff
@@ -0,0 +1,76 @@
+#
+# This is a Shiny web application. You can run the application by clicking
+# the 'Run App' button above.
+#
+# Find out more about building applications with Shiny here:
+#
+#    http://shiny.rstudio.com/
+#
+
+library(shiny)
+library(PEcAn.data.land)
+library(shinydashboard)
+library(dataone)
+
+#stopifnot
+
+# Define UI for application
+
+ui <- dashboardPage(
+  
+  dashboardHeader(title = "Data Ingest"),
+  dashboardSidebar(
+    sidebarMenu(
+      menuItem("Import Data", tabName = "importData", icon = icon("file")),
+      menuItem("Step 2 -- dbfiles record", tabName = "step2", icon = icon("cog")),
+      menuItem("Step 3 -- format record", tabName = "step3", icon = icon("cog")),
+      menuItem("Step 4 -- etc.", tabName = "step4", icon = icon("cog"))
+    )
+  ),
+  dashboardBody(
+    tabItems(
+      tabItem(tabName = "importData",
+              fluidRow( 
+                box(
+                  textInput("id", label = h3("Import From DataONE"), placeholder = "Enter doi or id here"),
+                  actionButton(inputId = "D1Button", label = "Upload"),
+                  hr(),
+                  fluidRow(column(12, verbatimTextOutput("identifier"))) 
+                ),
+                
+                box(
+                  # https://github.com/rstudio/shiny-examples/blob/master/009-upload/app.R
+                  fileInput(inputId = "file", label = h3("Upload Local Files"), accept = NULL, multiple = TRUE, placeholder = "Drag and drop files here"),
+                  p("This isn't linked to the server yet")
+                )
+              )
+      ),
+      
+      tabItem(tabName = "step2",
+              h2("dbfiles tab content")
+      )
+      
+      
+    )
+  )
+)
+
+server <- function(input, output) {
+  
+  # d1d <- eventReactive(input$D1Button, { input$id }) #print doi on click
+  
+  # How do I force R to load the dependencies before I run dataone_download? brew, redland, datapack, dataone
```

Comment:
R should handle it's own dependencies. Do I infer from this comment that it wasn't?

Reply:
It wasn't before, but I think it is now. I think this had something to do with the redland error... So I'll delete this comment now